drivers/encx24j600/encx24j600.c

```diff
@@ -96,12 +96,13 @@ void encx24j600_setup(encx24j600_t *dev, const encx24j600_params_t *params)
 static void encx24j600_isr(void *arg)
 {
     encx24j600_t *dev = (encx24j600_t *) arg;
+    netdev2_t *netdev = (netdev2_t *) arg;
```

Comment:
wouldn't it be simpler, if you cast `arg` only to `netdev2_t *` and use that instead of `dev->netdev.` below?